src/components/OSCALControlProse.js

```diff
@@ -162,15 +163,12 @@ function getImplReqSetParameters(implReqStatements, componentId) {
  * @param {String} key
  * @returns the text segment component
  */
-function getTextSegment(text, key) {
+function getTextSegment(text) {
   if (!text) {
     return null;
   }
-  return (
-    <Typography component="span" key={key}>
-      {text}
-    </Typography>
-  );
+
+  return <OSCALMarkupSingleLine text={text} />;
```

Comment:
I'm not sure how much of a concern this is at the moment, but by removing `Typography` and replacing it with `OSCALMarkupSingleLine`, it changes the text font of lines that have been given a description.
![Screenshot (9)](https://user-images.githubusercontent.com/81392411/183500459-7caaf54f-3d43-4655-8dc9-1255edf801b5.png)
![Screenshot (10)](https://user-images.githubusercontent.com/81392411/183500461-467efb23-51bd-4c41-bd04-fd3a763afd99.png)
The first screenshot is how it currently appears in `develop`, and the fonts are all the same. The second screenshot is how it appears when `Typography` is replaced with `OSCALMarkupSingleLine`, and the font for the lines that have been given descriptions is different from the rest.

Reply:
```suggestion
  return (
    <Typography component="span" key={key}>
      <OSCALMarkupSingleLine text={text} />
    </Typography>
  );
```

I believe this will fix the issue

Reply:
Thank you, I reinstated the Typography tags around the Markup component